profile/2019-03-01-hybrid.json

```diff
@@ -0,0 +1,236 @@
+                    "resourceGroups",
+                    "resources",
+                    "subscriptions",
+                    "subscriptions/locations",
+                    "subscriptions/operationresults",
+                    "subscriptions/providers",
+                    "subscriptions/resourceGroups",
+                    "subscriptions/resourceGroups/resources",
+                    "subscriptions/resources",
+                    "subscriptions/tagNames",
+                    "subscriptions/tagNames/tagValues",
+                    "tenants"
+            ]
+        },
+
+        "microsoft.storage": {
+
+            "2017-10-01": [
+                    "checkNameAvailability",
+                    "locations",
+                    "locations/quotas",
+                    "operations",
+                    "storageAccounts",
+                    "storageAccounts/blobServices",
+                    "storageAccounts/queueServices",
+                    "storageAccounts/tableServices",
+                    "usages"   
+            ]
+        },
+    
+        "microsoft.web": {
+
+            "2018-02-01": [
+                        "certificates",
+                        "operations",
+                        "checknameavailability",
+                        "sites",
+                        "sites/domainOwnershipIdentifiers",
+                        "sites/extensions",
+                        "sites/hostNameBindings",
+                        "sites/instances",
+                        "sites/instances/extensions",
+                        "sites/slots",
+                        "sites/slots/hostNameBindings",
+                        "sites/slots/instances",
+                        "sites/slots/instances/extensions",
+                        "serverFarms"
+            ],
+
+            "2016-09-01":[
+                        "serverFarms/metricDefinitions",
+                        "serverFarms/metrics",
+                        "serverFarms/usages"
+            ],
+
+            "2016-08-01": [
+                        "metadata",
+                        "sites/extensions",
+                        "sites/metricDefinitions",
+                        "sites/metrics",
+                        "sites/recommendations",
+                        "sites/slots/extensions",
+                        "sites/slots/metricDefinitions",
+                        "sites/slots/metrics"
+            ],
+
+            "2016-03-01": [
+                        "availableStacks",
+                        "deploymentLocations",
+                        "georegions",
+                        "ishostnameavailable",
+                        "isusernameavailable",
+                        "listSitesAssignedToHostName",
+                        "publishingUsers",
+                        "recommendations",
+                        "runtimes",
+                        "sourceControls",
+                        "validate"
+            ]
+        }
+
+    },
+
```

Comment:
can you please remove the extra lines and white spaces, just like other profile files?

Reply:
And please fix the indentation.